src/components/UsernameEntry/UsernameEntry.js

```diff
@@ -23,12 +23,17 @@ class UsernameEntry extends React.Component {
 					Please select your slack handle to send a reminder
 				</div>
 				<div>
-					<ButtonList
-						items={this.props.users}
-						handleClick={(id, name) => this.sendReminder(name)}
-					/>
+					{this.props.users.length !== 0 && (
+						<ListSelection
+							items={this.props.users.map(user => ({
```

Comment:
Would be good to extract this to a different method (ideally a selector could do this mapping so the component was only passing props)

e.g.

`items={usersWithAvatars}`